src/index.js

```diff
@@ -57,7 +59,28 @@ async function main(req, context) {
     });
   }
 
-  const mp = context.config.fstab.match(path);
+  let codeStorage;
+  let fstab;
+
+  try {
+    codeStorage = new AWSStorage({
+      AWS_S3_REGION,
+      AWS_S3_ACCESS_KEY_ID,
+      AWS_S3_SECRET_ACCESS_KEY,
+      bucket: 'helix-code-bus',
+      readOnly: true,
+      log,
+    });
+    const res = await codeStorage.load(`${owner}/${repo}/${ref}/fstab.yaml`);
+    fstab = await new MountConfig().withSource(await res.text()).init();
+  } finally {
```

Comment:
maybe log an error if the fstab could not be loaded?

Reply:
Added code for testing response from `load`